plugins/legacy-preset-chart-nvd3/src/utils/tokenize.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { validateNumber } from '@superset-ui/validator';
+
+const tokenizeToNumericArray = value => {
+  if (value && value.trim()) {
```

Comment:
`value.trim()` returns a new string and does not mutate `value`

```ts
if (value) {
  const tokens = value.trim().split(',').map(v => v.trim());
  ...
}
```

Reply:
The thought in the if-statement was merely to check that `value` is defined and isn't only whitespace. Could probably have been written more elegantly, will try to refactor more sensibly.

Reply:
I noticed that too, but the code doesn't actually depend on the value being trimmed. The `trim()` is just being used to check that it contains characters other than spaces, so I think this is fine.